packages/client/src/api/apply-default-execution-options-from-project.ts

```diff
@@ -1,53 +0,0 @@
-import { METICULOUS_LOGGER_NAME } from "@alwaysmeticulous/common";
-import { ReplayExecutionOptions } from "@alwaysmeticulous/sdk-bundles-api";
-import log from "loglevel";
-import { createClient } from "../client";
-import { getProject } from "./project.api";
-
-export const applyDefaultExecutionOptionsFromProject = async ({
-  apiToken,
-  executionOptions,
-}: {
-  apiToken: string | undefined | null;
-  executionOptions: ReplayExecutionOptions;
-}): Promise<ReplayExecutionOptions> => {
-  const logger = log.getLogger(METICULOUS_LOGGER_NAME);
-  const client = createClient({ apiToken });
-  const project = await getProject(client);
-  if (!project) {
-    throw new Error(
-      `Could not retrieve project data. Is the API token correct?`
-    );
-  }
-
-  if (
-    executionOptions.networkStubbingMode != null ||
-    project.settings.networkStubbingMode == null
-  ) {
-    return executionOptions;
-  }
-
-  if (project.settings.networkStubbingMode.type === "stub-non-ssr-requests") {
-    logger.info("");
```

Comment:
Note: we may want to keep this logger information somewhere, since I think it is useful feedback when using the CLI.

Reply:
This is moved to the bundles